optimizer_wrapper_test fail

When running following command:

```bash
pytest elasticdl/python/tests/optimizer_wrapper_test.py -k test_set_slot_values_to_variables
```

The test reports an error:

```
self = <tf.Tensor: id=51, shape=(3, 4), dtype=bool, numpy=
array([[ True,  True,  True,  True],
       [ True,  True,  True,  True],
       [ True,  True,  True,  True]])>

    def __bool__(self):
>     return bool(self._numpy())
E     ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

Here compares two ResourceVariable

```python
self.assertTrue(
                    slots_dict[slot]
                    == opt_wrapper._slot_variables[layer][slot]
                )
```

The results of `slots_dict[slot] == opt_wrapper._slot_variables[layer][slot]` is 
```
tf.Tensor(
[[ True  True  True  True]
 [ True  True  True  True]
 [ True  True  True  True]], shape=(3, 4), dtype=bool)
<class 'tensorflow.python.framework.ops.EagerTensor'>
```

The tf.tensor could not be asserted true directly.

--- a/elasticdl/python/tests/optimizer_wrapper_test.py
+++ b/elasticdl/python/tests/optimizer_wrapper_test.py
@@ -216,7 +216,10 @@ class OptimizerWrapperTest(unittest.TestCase):
                     ).all()
                 )
                 self.assertTrue(
-                    (slots_dict[slot].numpy() == opt_wrapper._slot_variables[layer][slot].numpy()).all()
+                    (
+                        slots_dict[slot].numpy()
+                        == opt_wrapper._slot_variables[layer][slot].numpy()
+                    ).all()
                 )
 
                 slots_dict[slot].assign(tf.ones((10, 4)))
@@ -271,7 +274,10 @@ class OptimizerWrapperTest(unittest.TestCase):
                 ).all()
             )
             self.assertTrue(
-                (grads_and_vars[i][1].numpy() == opt_wrapper._embed_variables[layer].numpy()).any()
+                (
+                    grads_and_vars[i][1].numpy()
+                    == opt_wrapper._embed_variables[layer].numpy()
+                ).any()
             )
 
         embedding_values_new = {"test-1": np.zeros((3, 4), np.float32)}

--- a/elasticdl/python/tests/optimizer_wrapper_test.py
+++ b/elasticdl/python/tests/optimizer_wrapper_test.py
@@ -216,8 +216,7 @@ class OptimizerWrapperTest(unittest.TestCase):
                     ).all()
                 )
                 self.assertTrue(
-                    slots_dict[slot]
-                    == opt_wrapper._slot_variables[layer][slot]
+                    (slots_dict[slot].numpy() == opt_wrapper._slot_variables[layer][slot].numpy()).all()
                 )
 
                 slots_dict[slot].assign(tf.ones((10, 4)))
@@ -264,6 +263,7 @@ class OptimizerWrapperTest(unittest.TestCase):
             grads_and_vars, embedding_values
         )
         for i, layer in enumerate(layers):
+            print(opt_wrapper._embed_variables[layer])
             self.assertTrue(
                 (
                     opt_wrapper._embed_variables[layer].numpy()
@@ -271,7 +271,7 @@ class OptimizerWrapperTest(unittest.TestCase):
                 ).all()
             )
             self.assertTrue(
-                grads_and_vars[i][1] == opt_wrapper._embed_variables[layer]
+                (grads_and_vars[i][1].numpy() == opt_wrapper._embed_variables[layer].numpy()).any()
             )
 
         embedding_values_new = {"test-1": np.zeros((3, 4), np.float32)}

--- a/elasticdl/python/tests/optimizer_wrapper_test.py
+++ b/elasticdl/python/tests/optimizer_wrapper_test.py
@@ -266,7 +266,6 @@ class OptimizerWrapperTest(unittest.TestCase):
             grads_and_vars, embedding_values
         )
         for i, layer in enumerate(layers):
-            print(opt_wrapper._embed_variables[layer])
             self.assertTrue(
                 (
                     opt_wrapper._embed_variables[layer].numpy()

--- a/elasticdl/python/tests/optimizer_wrapper_test.py
+++ b/elasticdl/python/tests/optimizer_wrapper_test.py
@@ -276,7 +276,7 @@ class OptimizerWrapperTest(unittest.TestCase):
                 (
                     grads_and_vars[i][1].numpy()
                     == opt_wrapper._embed_variables[layer].numpy()
-                ).any()
+                ).all()
             )
 
         embedding_values_new = {"test-1": np.zeros((3, 4), np.float32)}

